src/Microsoft.Health.Dicom.Core/Features/Validation/IDicomElementMinimumValidator.cs

```diff
@@ -9,6 +9,6 @@ namespace Microsoft.Health.Dicom.Core.Features.Validation
 {
     public interface IDicomElementMinimumValidator
     {
-        void Validate(DicomElement element);
+        void Validate(DicomElement dicomElement);
```

Comment:
We should probably add some XML doc here that indicates it throws `DicomElementValidationException`